src/test/java/net/revelc/code/impsort/ImpSortTest.java

```diff
@@ -201,4 +202,20 @@ public void testRemoveSamePackageImports() {
     assertTrue(imports.stream().anyMatch(imp -> "abcd.ef.Blah.Blah".equals(imp.getImport())));
     assertTrue(imports.stream().anyMatch(imp -> "abcd.efg.Blah2".equals(imp.getImport())));
   }
+
+  @Test
+  public void testResultStartWithComment() throws IOException {
+    Path p =
+            Paths.get(System.getProperty("user.dir"), "src", "test", "resources", "FirstImportComment.java");
+    Result result =
+            new ImpSort(StandardCharsets.UTF_8, eclipseDefaults, true, true, LineEnding.AUTO)
+                    .parseFile(p);
+
+    Path output = File.createTempFile("impSort", null).toPath();
```

Comment:
Please create temp files under the `/target` directory, rather than the system temp directory. This makes it easier to keep the responsibility for cleanup to the maven project, with `mvn clean`.